src/parser/paladin/holy/modules/azeritetraits/LightsDecree.js

```diff
@@ -68,7 +77,7 @@ class LightsDecree extends Analyzer {
     this.healingFromGlimmer += healingDone;
   }
 
-  on_beacontransfer(event) {
+  onBeaconTransfer(event) {
     if (!this.hasAvengingWrathBuffThroughLightsDecree(event)) {
       return;
     }
```

Comment:
Tracking this.healingFromGlimmer is a little misleading.  If you were to take some other talent instead of Light's Decree you would still be casing Holy Shocks and which would contribute to glimmer healing.  You would be casting half as many Holy Shocks in the 6.25 second interval when wings is not up if you didn't have Light's Decree.  Therefore I don't think its fair to count 100% of glimmer healing during the wings extension toward the traits contribution.  Additionally the effect of the extra glimmers persists for 30 seconds not the 6.25 second extension granted by Light's Decree.  The correct way to track how Light's Decree effects glimmer would be to calculate the number that would have been cast without light's decree and the quantity that was cast with it.  Then track all the glimmer healing done by glimmers that were applied during the Light's Decree window.  Then count a percentage of the total toward the bonus healing provided by the trait as some glimmers would still be applied without it.  Additionally you would want to count additional glimmer triggers for damage and healing.  For example if 1 Holy Shock cast could have been done in the window without Light's Decree but 3 Holy Shocks were cast in the same 6.25 second window with Light's Decree, you would want to count all the damage and healing from all glimmers toward the extra healing provided by the trait.